### DISCUSSION TITLE
Temperature drops 30 degrees after starting print

### DISCUSSION THREAD

**Discussion by AllanMartel** [mriscoc/Ender3V2S1#1315] [Q&A]
I have an Ender 3 v2 with a Sprite Pro and a CR Touch. I am running the latest firmware version created with the custom configurator set to use the T13 temperature sensor and input shaping on board 4.2.2. 

7 out of 10 times, the temperature drops right at the start of the print, upwards of 30 degrees. It does not do it on every print.

Example: my print is set at 205 degrees on the hotend. Right before the printer starts to print the prime strip, the temperature will suddenly drop to 175 degrees. I have to use the TUNE feature to bring the hotend temperature back up to 205. After this, it does not change again and finishes normally. The bed temperature does not change.

Any ideas? As of now, its just a minor annoyance, but if I can rectify this, I would like to.

**Comment by mriscoc** [comment]
Check your sliced G-code, maybe it has some temperature commands.
